src/Makefile

```
@@ -358,6 +367,7 @@ else
 endif
 
 ENGINE_NAME=valkey
+ENABLE_REDIS_SYMBLINK=true
```

```suggestion
CONFIGURE_REDIS_SYMLINKS?=true
```

?= so it can be overwritten. Also, I think CONFIGURE_REDIS_SYMLINKS is a bette name.

---

INSTALL_REDIS_SYMLINKS is the better name IMHO.